parsers/AW.py

```diff
@@ -55,17 +55,17 @@ def fetch_production(
     missing_from_total = reported_total - sources_total
     unknown = missing_from_total if missing_from_total > 0 else 0
     # We're using Fossil data to get timestamp in correct time zone
-    local_date_time = datetime.strptime(fossil["timestamp"], "%Y-%m-%d %H:%M:%S.%f")
+    local_date_time = datetime.strptime(biogas["timestamp"], "%Y-%m-%d %H:%M:%S.%f")
     zone_date_time = arrow.Arrow.fromdatetime(local_date_time, "America/Aruba")
 
     data = {
         "zoneKey": zone_key,
         "datetime": zone_date_time.datetime,
         "production": {
-            "oil": float(fossil["value"]),
-            "wind": float(wind["value"]),
-            "solar": float(solar["value"]),
-            "biomass": float(biogas["value"]),
+            "oil": round(float(fossil["value"]), 3),
+            "wind": round(float(wind["value"]), 3),
+            "solar": round(float(solar["value"]), 3),
+            "biomass": round(float(biogas["value"]), 3),
```

Comment:
If we want to round them and assume these are in MW we should round them to six decimals (1 W) not three (1kw). But I really don't see a need to round them. However, we might consider rounding the unknown one to six decimals as it's computed but it's also not really needed.
```suggestion
            "oil": round(float(fossil["value"]), 6),
            "wind": round(float(wind["value"]), 6),
            "solar": round(float(solar["value"]), 6),
            "biomass": round(float(biogas["value"]), 6),
```
or
```suggestion
            "oil": float(fossil["value"]),
            "wind": float(wind["value"]),
            "solar": float(solar["value"]),
            "biomass": float(biogas["value"]),
            "unknown": unknown,
```

Reply:
> If we want to round them and assume these are in MW, we should round them to six decimals (1 W) not three (1kw). But I really don't see a need to round them. However, we might consider rounding the unknown one to six decimals as it's computed but it's also not really needed or to thirteen decimals to get rid of floating-point errors as suggested by @q-- below.
> 
> or